### PR TITLE
Login Navigation Functioning Correctly && Signup Page Added

### PR DESCRIPTION
**Pull request by jamesramadan**

### CODE REVIEW

File: src/login/login.js [hunk 1]
```diff
@@ -24,25 +39,37 @@ export default class Login extends Component {
...
+                    placeholder={'Enter Username'}
                     maxLength={12}
                     multiline={false}
                     />
+                  <TextInput
+                      style={styles.input}
+                      value={this.state.password}
+                      onChangeText={(text) => this.setState({password: text})}
+                      placeholder={'Enter Password'}
+                      maxLength={12}
```
Comment (hannahhenderson) [inline src/login/login.js:51]: Add minlength?

File: src/login/login.js [hunk 2]
```diff
@@ -24,25 +39,37 @@ export default class Login extends Component {
...
+                  />
                  <TouchableHighlight
                     style={styles.button}
                     underlayColor={'#328FE6'}
-                    onPress={this.onPress.bind(this)}
+                    onPress= {() => this.onButtonPress()}
                     >
                     <Text style={styles.label}>LOGIN</Text>
                 </TouchableHighlight>
+                <Hyperlink>
```
Comment (hannahhenderson) [inline src/login/login.js:61]: Adjust indentation

File: src/login/signup.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  }
+
+  onPress() {
+    console.log(this.state.username);
+    this.props.navigator.push({
+      name: 'map'
+    })
+  }
+  
+  render() {
```
Comment (hannahhenderson) [inline src/login/signup.js:25]: Can any of this be refactored with the login page to make the code a bit more DRY?

**Comment by hannahhenderson** [comment]
A few minor comments, otherwise LGTM

File: src/login/login.js (hunk 1 above)
Comment (jamesramadan) [inline src/login/login.js:51]: Right now we'll leave it open for navigating around, but I agree we may need to add a restriction here.

File: src/login/login.js (hunk 2 above)
Comment (jamesramadan) [inline src/login/login.js:61]: Ok. Will do Thanks.

File: src/login/signup.js (hunk 3 above)
Comment (jamesramadan) [inline src/login/signup.js:25]: Probably. Will leave as is for now MVP.

**Pull request merged**